src/widgets/settings_widget.h

```diff
@@ -39,54 +39,54 @@ enum GameOptionsWidgets : WidgetID {
+	WID_GO_BASE_SFX_TEXTFILE,                ///< Open base SFX readme, changelog (+1) or license (+2).
+	WID_GO_BASE_SFX_DESCRIPTION = WID_GO_BASE_SFX_TEXTFILE + TFT_CONTENT_END, ///< Description of selected base SFX.
+	WID_GO_BASE_MUSIC_DROPDOWN,              ///< Use to select a base music set.
+	WID_GO_BASE_MUSIC_CONTENT_DOWNLOAD,      ///< 'Get Content' button for base music.
+	WID_GO_TEXT_MUSIC_VOLUME,                ///< Music volume label.
+	WID_GO_BASE_MUSIC_VOLUME,                ///< Change music volume.
+	WID_GO_BASE_MUSIC_JUKEBOX,               ///< Open the jukebox.
+	WID_GO_BASE_MUSIC_OPEN_URL,              ///< Open base music URL.
+	WID_GO_BASE_MUSIC_TEXTFILE,              ///< Open base music readme, changelog (+1) or license (+2).
 	WID_GO_BASE_MUSIC_DESCRIPTION = WID_GO_BASE_MUSIC_TEXTFILE + TFT_CONTENT_END, ///< Description of selected base music set.
-	WID_GO_VIDEO_ACCEL_BUTTON,     ///< Toggle for video acceleration.
-	WID_GO_VIDEO_VSYNC_BUTTON,     ///< Toggle for video vsync.
-	WID_GO_REFRESH_RATE_DROPDOWN,  ///< Dropdown for all available refresh rates.
-	WID_GO_VIDEO_DRIVER_INFO,      ///< Label showing details about the current video driver.
-	WID_GO_SURVEY_SEL,             ///< Selection to hide survey if no JSON library is compiled in.
-	WID_GO_SURVEY_PARTICIPATE_BUTTON, ///< Toggle for participating in the automated survey.
-	WID_GO_SURVEY_LINK_BUTTON,     ///< Button to open browser to go to the survey website.
-	WID_GO_SURVEY_PREVIEW_BUTTON,  ///< Button to open a preview window with the survey results
-	WID_GO_SOCIAL_PLUGINS,         ///< Main widget handling the social plugins.
-	WID_GO_SOCIAL_PLUGIN_TITLE,    ///< Title of the frame of the social plugin.
-	WID_GO_SOCIAL_PLUGIN_PLATFORM, ///< Platform of the social plugin.
-	WID_GO_SOCIAL_PLUGIN_STATE,    ///< State of the social plugin.
+	WID_GO_VIDEO_ACCEL_BUTTON,               ///< Toggle for video acceleration.
+	WID_GO_VIDEO_VSYNC_BUTTON,               ///< Toggle for video vsync.
+	WID_GO_REFRESH_RATE_DROPDOWN,            ///< Dropdown for all available refresh rates.
+	WID_GO_VIDEO_DRIVER_INFO,                ///< Label showing details about the current video driver.
+	WID_GO_SURVEY_SEL,                       ///< Selection to hide survey if no JSON library is compiled in.
+	WID_GO_SURVEY_PARTICIPATE_BUTTON,        ///< Toggle for participating in the automated survey.
+	WID_GO_SURVEY_LINK_BUTTON,               ///< Button to open browser to go to the survey website.
+	WID_GO_SURVEY_PREVIEW_BUTTON,            ///< Button to open a preview window with the survey results
+	WID_GO_SOCIAL_PLUGINS,                   ///< Main widget handling the social plugins.
+	WID_GO_SOCIAL_PLUGIN_TITLE,              ///< Title of the frame of the social plugin.
+	WID_GO_SOCIAL_PLUGIN_PLATFORM,           ///< Platform of the social plugin.
+	WID_GO_SOCIAL_PLUGIN_STATE,              ///< State of the social plugin.
```

Comment:
No need for this.

Reply:
What should the style be? The file itself uses an inconsistent spacing. I also didn't see anything in the coding style doc  about it.